app/views/pages/partials/_footer.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= vite_image_tag "images/wave-green.png", id: "wave" %>
+
+<footer>
+
+  <%= vite_image_tag "images/logos/logo_text_black_clipped.png", id: "black-logo", class: "f-deco" %>
+  <%= vite_image_tag "images/illustrations/1010.png", id: "plant-bottom", class: "f-deco" %>
+
+  <div class="footer-container">
+    <div class="footer-content">
+
+      <nav>
+        <ul>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Sessions</a></li>
+          <li><a href="#">Videos</a></li>
+          <li><a class="btn-contact" href="#">Contact me</a></li>
+        </ul>
+      </nav>
+
+      <div class="content-socials">
+        <h4>Follow me:</h4>
+        <a href="#"><i class="fa-brands fa-facebook"></i></a>
+        <a href="#"><i class="fa-brands fa-instagram"></i></a>
+      </div>
+
+    </div>
+  </div>
+
+  <h4 class="content-copyright">Copyright © YogaUnalome 2023. All rights reserved.</h4>
+
+</footer>
```

Comment:
The wave should be inside the footer.
You can also remove the empty spaces between parent elements and child elements. The indentation makes it readable enough :) That means line 4, 10, 25 and 30. The rest of your spacing is nice.

Your nav should not have a ul or li
It nav with anchors tags directly inside
```
nav
  a
  a
  a
/nav
```

Also the video page will not be done before a while so you can remove this for now.

Reply:
all done